### PR TITLE
Fragmentation can now hit roaches

### PR DESCRIPTION
**Pull request by Mocha-Not-Latte**
## About The Pull Request
As the title says, a modular change to allow fragments from Pommes and Stingers now can hit roaches and actually hurt them instead of being useless.
Fragment

Yes, I know code freeze still 

## Why It's Good For The Game
Makes fragment grenades actually useful against roaches. 

## Changelog
```changelog
tweak: Roaches are now hit by fragmentation 
balance: Stingers do a minor amount of brute; they're projectiles being shot at you, they should. They've also been toned down agony wise.
```

### CODE REVIEW

**Comment by RadiantFlash7** [comment]
I'm really happy you separated this change from the rest of the GL rework PR, because this is something that really should be changed, whereas I'm a bit more hesitant on the launcher changes.

**Comment by ShadzFen** [comment]
Looks fine, but I'm curious why you toned down the agony damage for the stingers? Is it too high?

File: zzzz_modular_occulus/code/game/objects/items/weapons/grenades/stinger.dm [hunk 1]
```diff
@@ -8,9 +8,9 @@
 	item_state = "frggrenade"
 	loadable = TRUE
 
-	fragment_type = /obj/item/projectile/bullet/pellet/shotgun/rubber/stinger
+	fragment_type = /obj/item/projectile/bullet/pellet/fragment/rubber/stinger
 	num_fragments = 100  //total number of fragments produced by the grenade
-	fragment_damage = 0
+	fragment_damage = 2
```
Comment (Jamini) [inline zzzz_modular_occulus/code/game/objects/items/weapons/grenades/stinger.dm:13]: This is going to cause 150-200 damage to someone on the same tile as a stinger

File: zzzz_modular_occulus/code/modules/projectiles/projectile/bullettypes.dm [hunk 2]
```diff
@@ -11,15 +11,19 @@
...
+	icon_state = "birdshot-1"
 	damage_types = list(BRUTE = 2)
-	agony = 18
-	pellets = 4
-	base_spread = 0 //causes it to be treated as a shrapnel explosion instead of cone
-	spread_step = 20
-	silenced = 1 //embedding messages are still produced so it's kind of weird when enabled.
-	no_attack_log = 1
-	muzzle_type = null
+	agony = 10
```
Comment (Jamini) [inline zzzz_modular_occulus/code/modules/projectiles/projectile/bullettypes.dm:18]: This is a reduction from 18 (1800 total) agony to 12 agony (1000 from agony, 200 from brute.) Why are you reducing this?

**Review by Jamini — CHANGES_REQUESTED** [review]

**Comment by Jamini** [comment]
Please provide reasoning for reducing the overall agony from a stinger, as well as justifying sufficient brute damage to instantly put someone into critical health (and potentially death)

File: zzzz_modular_occulus/code/modules/projectiles/projectile/bullettypes.dm (hunk 2 above)
Comment (Mocha-Not-Latte) [reply zzzz_modular_occulus/code/modules/projectiles/projectile/bullettypes.dm]: > 

Brute toned down to 1 per fragment.
Agony up to 17 agony per.

**Review by Jamini — APPROVED** [review]
Thank you Mocha. I look forward to usable stingers.

**Pull request merged**